tests/forms_tests/field_tests/test_datefield.py

```diff
@@ -109,7 +109,7 @@ def test_l10n_invalid_date_in(self):
     def test_form_label_association(self):
         # label tag is correctly associated with first rendered dropdown
         a = GetDate({'mydate_month': '1', 'mydate_day': '1', 'mydate_year': '2010'})
-        self.assertIn('<label for="id_mydate_day">', a.as_p())
+        self.assertIn('<label for="id_mydate_day"', a.as_p())
```

Comment:
I don't think that this change should be necessary. Is there something wrong in the template?